banklib/src/lib.rs

```diff
@@ -0,0 +1,189 @@
+use std::io::{Read, Write};
+use std::net::TcpStream;
+
+use protocol_crate::{Command, Response, BankError, Operation};
+
+const SERVER_ADDRESS: &str = "127.0.0.1:7878";
+
+pub struct BankClient {
+    server_address: String,
+}
+
+impl BankClient {
+    pub fn new(x: &str) -> Self {
+        BankClient {
+            server_address: x.to_string(),
+        }
+    }
+
+    /// Creates a new account with the given `account` name.
+    ///
+    /// # Arguments
+    ///
+    /// * `account` - The name of the account to be created.
+    ///
+    /// # Returns
+    ///
+    /// * `Ok(usize)` - The ID of the newly created account.
+    /// * `Err(BankError)` - If the account already exists or there was an error during the process.
+    ///
+    pub fn create_account(&self, account: String) -> Result<usize, BankError> {
+        let response = self.send_command(Command::CreateAccount(account));
+        match response {
+            Response::Account(result) => Ok(result?),
```

Comment:
Можно так оставить, а можно сделать покороче:

```suggestion
            Response::Account(result) => result,
```